plugins/plugin-languageserver/che-plugin-languageserver-ide/pom.xml

```diff
@@ -149,7 +149,7 @@
                 <version>${project.version}</version>
                 <executions>
                     <execution>
-                        <phase>process-sources</phase>
+                        <phase>generate-sources</phase>
```

Comment:
It's impossible to build in offline mode when the local maven cache is empty. I can remove che stuff from the cache, though.